docs/index.md

```diff
@@ -1,26 +1,28 @@
-# Welcome to the EasyBuild documentation!
+# EasyBuild documentation
 
-!!! warning
-    This is the new EasyBuild documentation,
-    made with [Material for MkDocs](https://squidfunk.github.io/mkdocs-material)
-    and written in [MarkDown](https://daringfireball.net/projects/markdown).
+![EasyBuild logo](img/easybuild_logo_2022_vertical_dark_bg_transparent.png){: align=left style="height:455px;width:400px"}
 
-    We intend to replace the <https://docs.easybuild.io>,
-    which is made with [Sphinx](https://www.sphinx-doc.org)
-    and written in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText)
-    with this documentation soon...
+Welcome to the documentation of **[EasyBuild](https://easybuild.io) (version {{ easybuild.version }})**.
 
-    **
-    If you want to help with porting the EasyBuild documentation to MarkDown, you can!  
-    Please see <https://github.com/easybuilders/easybuild-docs>.
-    **
+Use the menu to navigate, or use the search box (top right).
```

Comment:
```suggestion
Use the menu to navigate, or use the search (top right).
```

It is not a box when on a small screen, but it is still in the top right.

Reply:
I'll use `search feature` instead, but good suggestion